### PR TITLE
libcurl: support for with-ca-fallback option

### PR DESCRIPTION
**Pull request by PodnimatelPingvinov**
Just one more option for libcurl package.

### CODE REVIEW

File: recipes/libcurl/all/conanfile.py [hunk 1]
```diff
@@ -597,6 +602,9 @@ def _generate_with_cmake(self):
         else:
             tc.cache_variables["CURL_CA_PATH"] = "none"
 
+        if self.options.with_ca_fallback:
+            tc.cache_variables["CURL_CA_FALLBACK"] = True
```
Comment (SpaceIm) [inline recipes/libcurl/all/conanfile.py:606]:
```suggestion
        tc.cache_variables["CURL_CA_FALLBACK"] = self.options.with_ca_fallback
```

**Review by SpaceIm — CHANGES_REQUESTED** [review]
Prefer to have full control over options instead of relying implicitly on default upstream values.

**Comment by PodnimatelPingvinov** [comment]
> Prefer to have full control over options instead of relying implicitly on default upstream values.

Sure, that's make sense!

**Review by Hopobcn — APPROVED** [review]

**Review by prince-chrismc — APPROVED** [review]

**Review by jwillikers — APPROVED** [review]

**Comment by SpaceIm** [comment]
PR should be closed/open to trigger CI again. openssl packages were missing, and this recipe must pass v2 pipeline.

**Comment by PodnimatelPingvinov** [comment]
> PR should be closed/open to trigger CI again. openssl packages were missing, and this recipe must pass v2 pipeline.

I'm not sure about CI pipelines so should I rebase this PR on master and then close/open it or rebase isn't needed?

**Comment by prince-chrismc** [comment]
No need for a rebase.

**Pull request merged**